README.md

```diff
@@ -6,11 +6,14 @@ This repository contains a sample banking application built using Cloud Spanner.
 
 NOTE: Requires gcloud, mvn, grpc_cli installed.
 
-1. Create a database locally using cloud-spanner-emulator and export spanner host
+1. Create a database locally using [cloud-spanner-emulator](https://cloud.google.com/spanner/docs/emulator) and export spanner host
 for client libraries to work.
 
     ```
     $ gcloud emulators spanner start
```

Comment:
This looks mildly misleading. At least the way I first read this made it seem like this singular line is what did the "Create a database locally using cloud-spanner-emulator and export spanner host for client libraries to work" since this line is now separated from the rest of the code that was originally under this statement. Maybe it would seem a bit clearer if you did something like:

Create a database locally using cloud-spanner-emulator and export spanner host for client libraries to work.
Begin by starting an emulator in one terminal window.